### ISSUE TITLE
How to transform the training data's annotations to suit your project?

### ISSUE THREAD

**Issue by Bigwode** [JakeRenn/pose_estimation#3]
Hello, Thanks for your project first ! But when I looked through your code and I had no idea of how to transform the original .mat annotations to .txt as the training data's labels to suit your project. So, can you share your skills in how to do this work or share your training data's labels by your convenient.Thanks a lot !

**Comment by JakeRenn** [comment]
@Bigwode 

The following codes are used for reading .txt file, lying in the `read_data.py` file.
```python
        with open(annos_path, 'rb') as fr:
            for line in fr:
                tmp = re.split(',| ', line.strip())
                if(len(tmp) != self.record_len):
                    print "Length Error: ", len(tmp)
                    sys.exit(0)
                filename = tmp[0]
                coords = [int(x) for x in tmp[1:self.record_len - 1]]
                begin = int(tmp[-1])
                self.records.append((filename, np.array(coords), begin))
```

So, the data format is like this
`        filename x y x y x y ..... 1 or 0 for begin`
The first item in each line denotes the filename of the corresponding image. And the last item in each line denotes whether it is the first image of a sequence of images from a video.
You can also customize the above code for your own use.

**Comment by Bigwode** [comment]
Thanks for you reply! But my question is How to transform the dataset of MPII's annotations to suit your project , because you don't share your training data but your training labels.So when I run your demo,it has a wrong result because your demo has little pictures.

**Comment by JakeRenn** [comment]
@Bigwode 

The script for transforming MPII's annotations from Matlab format to others is not provided here. And I did not write any script for this. 

This is a simplified version of the original project. The whole project may be released later and detailed document about it will be provided. 

By the way, the network architecture in this repo is too simple for MPII dataset. You may try CPM and Hourglass Network, which is powerful and suitable for human pose estimation. I did implement these network with TF and Pytorch. I will upload my code if it is needed.

**Comment by Bigwode** [comment]
Oh,thanks a lot!!! Amazing work that you implement the CPM and Hourglass Network with TF, please share this for your convenience! Enjoy it!
